extras/linked-token/README.md

```diff
@@ -1,66 +1,200 @@
-## Foundry
+# Linked Token Readme
 
-**Foundry is a blazing fast, portable and modular toolkit for Ethereum application development written in Rust.**
+## Outline for Linked Token Bridge
+- Contracts
+    - ERC20 token named USDCTest on Calibration Network
+    - Token Controller contract on Calibration Network
+    - Token Replica contract on subnet
+- Workflow
+    - Deposit tokens from rootnet to subnet
+        - calling the rootBridge.bridgeToken() to lock up USDC tokens and initiate an IPC cross net message to the subnet token bridge
+        - subnet token bridge receives IPC message and mints USDC proxy tokens
+    - Withdraw tokens from subnet to rootnet
+        - call subnetBridge.withdrawToken() to burn proxy USDC tokens on the subnet and initiate an IPC cross net message to the root network bridge
 
-Foundry consists of:
+## USDC propagation flow
```

Comment:
There is nothing in these contracts tying them to USDC, not sure why the insistence here in USDC?